From b398452c6f51795beed1f44ab2e575fc81883be8 Mon Sep 17 00:00:00 2001
From: czou <cao.zou@windriver.com>
Date: Mon, 24 Aug 2015 14:44:52 +0800
Subject: [PATCH 25/29] mmc: omap3-evm: add mmc cd gpio

mmc using this gpio to detect the plug/uplug of sdcard

Signed-off-by: czou <cao.zou@windriver.com>
---
 arch/arm/boot/dts/omap3-evm.dts | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/arch/arm/boot/dts/omap3-evm.dts b/arch/arm/boot/dts/omap3-evm.dts
index e7246e7..9b04dc2 100644
--- a/arch/arm/boot/dts/omap3-evm.dts
+++ b/arch/arm/boot/dts/omap3-evm.dts
@@ -93,3 +93,16 @@
 &usbhsehci {
 	phys = <0 &hsusb2_phy>;
 };
+
+&mmc1 {
+	cd-gpios = <&twl_gpio 0 GPIO_ACTIVE_HIGH>;
+	wp-gpios = <&gpio2 31 GPIO_ACTIVE_HIGH>;
+};
+
+&mmc2 {
+	status = "disabled";
+};
+
+&mmc3 {
+	status = "disabled";
+};
-- 
1.9.1

